openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml

```
@@ -0,0 +1,53 @@
+type: object
+properties:
+  data:
+    type: array
+    items:
+      type: object
+      properties:
+        period:
+          description: Revenue month.
+          type: string
+          pattern: '^\d{4}-\d{2}$'
+        total:
+          description: Total Monthly Recurring Revenue (MRR) amount.
+          type: number
+          format: double
+        currency:
+          $ref: ./CurrencyCode.yaml
+        breakdown:
+          description: |
+            MRR categories are described below.
+            The difference between current period MRR and previous period MRR is equal to `new + reactivated + expanded - contracted - churned`.
+          type: object
+          properties:
+            new:
+              description: |
+                MRR of new customers.
+                Example: A new customer signs up to a $40 per month plan. This is $40 in new MRR.
+              type: number
+              format: double
+            reactivation:
+              description: |
+                A previous customer stops being a customer for a period.
+                The same customer signs up again and purchases a $25 per month plan. This is reactivation MRR of $25.
+              type: number
+              format: double
+            churned:
```

Should this also be renamed? @adamaltman @noon-dawg 
```suggestion
            churn:
```

---

No, this is correct as churned.